meetings/security/2016-12-14/minutes.md

```diff
@@ -1,19 +1,63 @@
-# Security stream meeting 1
+# Swift Server APIs: Security stream meeting 1
 
-### Time: Wednesday Dec 14th, 2016
-* 6pm CET 
-* 5pm UK 
-* 12pm EST 
-* 9am PST
+### Initial Agenda:
+* Current discussion status roundup
+* Review of BlueSSLService, a layer on top of OpenSSL and macOS Secure Transport libraries:  BlueSSLService Review
+* Next Steps
 
-### WebEx and Agenda/Minutes doc:
-* WebEx:     https://meetings.webex.com/collabs/#/meetings/detail?uuid=M6T97FS5EMMK5RKXHFH0P5V2MR-1T5J&rnd=352264.78890
-* Agenda:    https://docs.google.com/document/d/168o2pYw4IBwv93bd61-_Y5y91GLxFpHxQ29gmExx_zs/edit?usp=sharing
+### Attendees:
+* Chris Bailey
+* David Sperling
+* Gelareh Taban
+* Alfredo Delli Bovi
+* Bill Abt
+* Alex Blewitt
```

Comment:
I'm not sure there's much benefit to listing the attendees alphabetically, so I'll leave as-is (we've never consciously listed alphabetically in the past!).